Automattic/wp-calypso
# Create product card for i2 version of pricing page

### Changes proposed in this Pull Request

This PR implements the new product card for the last iteration of the pricing page. See board for details and mockups.

Fixes 1196341175636977-as-1198514641268269

### Implementation notes

- I created a new chain of components (card + features section) altogether, similarly to the other versions.
- Text in the ribbon is dynamic to support internationalization and copy updates.
- The features section will be tackled in a separate PR.

### Testing instructions

- Download the PR locally
- Run Calypso and cloud concurrently
- Make sure the variant `v2 - slide outs` is selected in the `jetpackConversionRateOptimization` A/B test
- Visit the plans page in both Calypso and cloud, and compare it to the mockups (or screenshots below)

### Screenshots

#### Calypso
_Mockups_
<img width="290" alt="Screen Shot 2020-10-26 at 10 38 14 AM" src="https://user-images.githubusercontent.com/1620183/97190797-92ba5280-177c-11eb-8b35-285a39a29a1c.png">

_Implementation_
<img width="358" alt="Screen Shot 2020-10-26 at 11 07 16 AM" src="https://user-images.githubusercontent.com/1620183/97190852-a2399b80-177c-11eb-9875-e2a639b884a4.png">
<img width="358" alt="Screen Shot 2020-10-26 at 11 07 22 AM" src="https://user-images.githubusercontent.com/1620183/97190849-a2399b80-177c-11eb-84ac-ee4a8c45d0eb.png">


#### Cloud
_Mockups_
<img width="289" alt="Screen Shot 2020-10-26 at 10 37 57 AM" src="https://user-images.githubusercontent.com/1620183/97190775-8930ea80-177c-11eb-9d1a-78b91bdb91fe.png">

_Implementation_
<img width="355" alt="Screen Shot 2020-10-26 at 11 06 20 AM" src="https://user-images.githubusercontent.com/1620183/97190826-9a79f700-177c-11eb-80d2-18cd30824bb5.png">
<img width="359" alt="Screen Shot 2020-10-26 at 11 06 26 AM" src="https://user-images.githubusercontent.com/1620183/97190824-99e16080-177c-11eb-9301-185fa25f6879.png">
